### PR TITLE
WL-3596 : Remove max height and allow height to shrink with image.

### PR DESCRIPTION
**Pull request by nicholaswilson100**

### CODE REVIEW

**Comment by nicholaswilson100** [comment]
Linked with https://github.com/ox-it/wl-portal/pull/36

File: library/src/webapp/skin/neo-ox/portal.css [hunk 1]
```diff
@@ -830,17 +830,24 @@ ul#otherSiteList li a span, ul.otherSitesCategorList li a span {
     -moz-box-shadow:    0 2px 2px 0 rgba(0, 0, 0, 0.2);
     -webkit-box-shadow: 0 2px 2px 0 rgba(0, 0, 0, 0.2);
     box-shadow:         0 2px 2px 0 rgba(0, 0, 0, 0.2);
+    position: relative;
+}
+
+#siteBanner div:first-child {
+	position:absolute;
+	top:50%;
+	transform:translate(0, -50%);
```
Comment (buckett) [inline library/src/webapp/skin/neo-ox/portal.css]: I don't think transform is supported on IE8.

Comment (nicholaswilson100) [inline library/src/webapp/skin/neo-ox/portal.css]: You're right.  Also, it's clear that Firefox extensions like User Agent Switcher which are supposed to simulate other browsers don't really work.  Using this extension, it all looks peachy ; but if you look at it in an actual instance of IE8 (actually I'm looking at it in PlayonLinux) then the transform is ignored.

Do we have anything like an 'ie8.css' file that gets loaded if it's IE8?

Comment (buckett) [inline library/src/webapp/skin/neo-ox/portal.css]: I don't think there is an IE8 css file and generally I think it's better to try not to have browser specific hack as it makes support a little more complex. Is there no way todo this without using transform?

Comment (nicholaswilson100) [inline library/src/webapp/skin/neo-ox/portal.css]: I tried a few ways unsuccessfully (vertical-align:middle, align:middle/centre, etc.) and happened on this but vertical align is always a bit of a pain when you've got floating elements that can change in height.  There are some other ways which I'll try.....

Comment (buckett) [inline library/src/webapp/skin/neo-ox/portal.css]: Do the elements need to float?

Comment (nicholaswilson100) [inline library/src/webapp/skin/neo-ox/portal.css]: Well they already are (or were) float:left and float:right (which obviously I can change) but normally when you have one element on the left and the other at the same level on the right (that you're positioning wrt the rhs) then  it is good practice to do float:left and float:right.

Comment (nicholaswilson100) [inline library/src/webapp/skin/neo-ox/portal.css]: That said, it's defintiely v undesirable to have an IE8 hack but then it might not be an either or

Comment (buckett) [inline library/src/webapp/skin/neo-ox/portal.css]: You need the `ms-` prefix for IE9 support as well and the `webkit-` prefix for Safari.
http://caniuse.com/#search=transform

**Comment by nicholaswilson100** [comment]
Thanks. using margin-top rather than transform which was causing a lot of cross browser pain...It might mean it might not look fantastic for narrow browsers but it's a better solution ..

**Pull request merged**